invokeai/app/invocations/image_to_latents.py

```diff
@@ -44,12 +46,17 @@ class ImageToLatentsInvocation(BaseInvocation):
         input=Input.Connection,
     )
     tiled: bool = InputField(default=False, description=FieldDescriptions.tiled)
+    # NOTE: tile_size = 0 is a special value. We use this rather than `int | None`, because the workflow UI does not
+    # offer a way to directly set None values.
+    tile_size: int = InputField(default=0, multiple_of=8, description=FieldDescriptions.vae_tile_size)
```

Comment:
We've lucked out with this particular change not requiring a true `None` value. At some point, we'll need to add that capability.